packages/cozy-client/src/models/permission.js

```diff
@@ -0,0 +1,139 @@
+import intersection from 'lodash/intersection'
+import get from 'lodash/get'
+import CozyClient from 'cozy-client'
+import { getParentFolderId } from './file'
+
+/**
+ * @typedef {object} Document - Couchdb document like an io.cozy.files
+ * @property {string} _id
+ * @property {string} _type
+ */
+
+/**
+ * @typedef {('ALL'|'GET'|'PATCH'|'POST'|'PUT'|'DELETE')} PermissionVerb
+ */
+
+/**
+ * @typedef {object} PermissionItem
+ * @property {PermissionVerb[]} verbs - ALL, GET, PUT, PATCH, DELETE, POST…
+ * @property {string} selector - defaults to `id`
+ * @property {string[]} values
+ * @property {string} type - a couch db database like 'io.cozy.files'
+ */
+
+/**
+ * Is this permission read only ?
+ *
+ * @private
+ * @param {PermissionItem} perm - permission node in a io.cozy.permissions document
+ * @param {object} options -
```

Comment:
Nit: Throughout this file, there are end of line dashes, I think it's best to remove them.

Reply:
I've  put them toremove prettier warnings about not having descriptions for these param (and well, don't add useless descriptions)